src/assets/translations/en-US.json

```
@@ -2657,7 +2657,7 @@
       "Provisioning Criteria": "Provisioning Criteria",
       "Purchase Price Ratio": "Purchase Price Ratio",
       "Purchased/Redeemed Price": "Purchased/Redeemed Price",
```

Biome (2.4.15)

[error] The key Purchased/Redeemed Price was already declared.

(lint/suspicious/noDuplicateObjectKeys)